MapEditor.py

```diff
@@ -121,6 +122,17 @@ def _recompile(self):
             # TODO: make work...
             pack_map(self.dst_tv.get_filepath(sid))
 
+    def _save(self):
+        selection = op.relpath(self.dst_tv.get_filepath(self.dst_tv.selection()), self.paths['ROMFS_PATCH'])
+        packed_dir = op.dirname(selection)
```

Comment:
you shouldn't do this manually. Use `os.path.join(~)` to join strings to form a path

Reply:
This comment may have been attached to the wrong line, but it refers to the multiple lines below (128, 129 and 132)

Reply:
Alright, I changed that! Let me know if there's anything else to do.